VirtualTest/Test2/Test2.ino

```diff
@@ -0,0 +1,360 @@
+                    {'W',' ','W',' ','.',' ','.',' ','.',' ','W'},
+                    {'.','W','.','W','.','W','.','W','.','W','.'}};
+
+void printVirtualMap() {
+  for(int iX = 0 ; iX < iSize ; iX++) {
+    for(int iY = 0 ; iY < iSize ; iY++) {
+      Serial.print(virtualMap[iX][iY]);
+      if(iY < iSize - 1) Serial.print(" ");
+    }
+    Serial.println("");
+  }
+}
+
+void moveMapNorth() {
+  virtualMap[iRow][iColumn] = 'V';
+  virtualMap[iRow - 2][iColumn] = 'R';
+  iRow -= 2;
+}
+
+void moveMapEast() {
+  virtualMap[iRow][iColumn] = 'V';
+  virtualMap[iRow][iColumn + 2] = 'R';
+  iColumn += 2;
+}
+
+void moveMapSouth() {
+  virtualMap[iRow][iColumn] = 'V';
+  virtualMap[iRow + 2][iColumn] = 'R';
+  iRow += 2;
+}
+
+void moveMapWest() {
+  virtualMap[iRow][iColumn] = 'V';
+  virtualMap[iRow][iColumn - 2] = 'R';
+  iColumn -= 2;
+}
+
+void printMap(Map tiles_map, const uint8_t iFloor) {
+  char char_map[tiles_map.getRows()*2 + 1][tiles_map.getColumns()*2 + 1];
+  int row_char, column_char;
+
+  for(int i = 0; i < tiles_map.getRows()*2 + 1; ++i) {
+    for(int j = 0; j < tiles_map.getColumns()*2 + 1; ++j) {
+      char_map[i][j] = '.';
+    }
+  }
+  
+  row_char = 1;
+
+  for (int row_tile = 0 ; row_tile < tiles_map.getRows() ; row_tile++) {
+    column_char = 1;
+    for (int column_tile = 0 ; column_tile < tiles_map.getColumns() ; column_tile++) {
+      if(tiles_map.getTile(row_tile, column_tile).getStart()) {
+        char_map[row_char][column_char] = 'S';
+      }
+      else if(tiles_map.getTile(row_tile, column_tile).getVisited()) {
+        char_map[row_char][column_char] = 'V';
+      }
+      else if(tiles_map.getTile(row_tile, column_tile).getAccessible()) {
+        char_map[row_char][column_char] = ' ';
+      }
+      else {
+        char_map[row_char][column_char] = 'X';
+      }
+      if(tiles_map.getTile(row_tile, column_tile).getNorth()) {
+        char_map[row_char - 1][column_char] = '.';
+      }
+      else {
+        char_map[row_char - 1][column_char] = 'W';
+      }
+      if(tiles_map.getTile(row_tile, column_tile).getEast()) {
+        char_map[row_char][column_char + 1] = '.';
+      }
+      else {
+        char_map[row_char][column_char + 1] = 'W';
+      }
+      if(tiles_map.getTile(row_tile, column_tile).getSouth()) {
+        char_map[row_char + 1][column_char] = '.';
+      }
+      else {
+        char_map[row_char+1][column_char] = 'W';
+      }
+      if(tiles_map.getTile(row_tile, column_tile).getWest()) {
+        char_map[row_char][column_char - 1] = '.';
+      }
+      else {
+        char_map[row_char][column_char - 1] = 'W';
+      }
+      column_char += 2;
+    }
+    row_char += 2;
+  }
+  
+  for(int i = 0 ; i < tiles_map.getRows()*2 + 1 ; i++) {
+    for(int j = 0 ; j < tiles_map.getColumns()*2 + 1 ; j++) {
+      Serial.print(char_map[i][j]);
+      Serial.print(" ");
+    }
+    Serial.println(" ");
+  }
+  Serial.println(" ");
+}
+
+struct coord {
+  uint8_t x, y;
+};
```

Comment:
Or use coord instead of current_row and current_col

Reply:
Ok, is it fine if I change it after the regional tournament?